rfcs/rfc0015.md

```diff
@@ -0,0 +1,134 @@
+# Remove apostrophe ("Old package separator") as package separator
+
+## Preamble
+
+    Author:  Nicolás Mendoza <mendoza@pvv.ntnu.no>
+    Sponsor:
+    ID:      NICOMEN
+    Status:  Draft
+
+## Abstract
+
+Remove support for ' (apostrophe) as package namespace separator.
+
+## Motivation
+
+* Removal of ambiguous syntax that complicates parsing and confuses users.
+
+From perldoc perlmod
+
+```The old package delimiter was a single quote, but double colon is now the preferred delimiter, in part because it's more readable to humans, and in part because it's more readable to emacs macros. It also makes C++ programmers feel like they know what's going on--as opposed to using the single quote as separator, which was there to make Ada programmers feel like they knew what was going on. Because the old-fashioned syntax is still supported for backwards compatibility, if you try to use a string like "This is $owner's house", you'll be accessing $owner::s; that is, the $s variable in package owner, which is probably not what you meant. Use braces to disambiguate, as in "This is ${owner}'s house".```
```

Comment:
Markdown formatting point here: You probably wanted to use blockquote `>` rather than these triple-backticks

    > The old package delimiter was ...